src/mono/mono/mini/interp/transform-simd.c

```diff
@@ -387,6 +408,37 @@ emit_sri_vector128 (TransformData *td, MonoMethod *cmethod, MonoMethodSignature
 			simd_opcode = MINT_SIMD_INTRINS_P_PP;
 			simd_intrins = INTERP_SIMD_INTRINSIC_V128_AND_NOT;
 			break;
+		case SN_As:
+		case SN_AsByte:
+		case SN_AsDouble:
+		case SN_AsInt16:
+		case SN_AsInt32:
+		case SN_AsInt64:
+		case SN_AsNInt:
+		case SN_AsNUInt:
+		case SN_AsPlane:
+		case SN_AsQuaternion:
+		case SN_AsSByte:
+		case SN_AsSingle:
+		case SN_AsUInt16:
+		case SN_AsUInt32:
+		case SN_AsUInt64:
+		case SN_AsVector: {
+			return FALSE;
```

Comment:
@kg, I don't see any obvious existing handlers for what is effectively "bitcast" (that is the input is returned directly with no change, the API only exists to satisfy the type system).

Do you have a pointer to any similar handling that might exist?

Reply:
simd-methods.def and transform-simd are where most of the supported names are, iirc. i don't see support for bitcast and don't remember implementing it.

Reply:
I ended up following what `op_UnaryNegation` does and adding handlers that basically do memcpy to handle it.